### PR TITLE
FAQ - Rapid tests / Schnelltests

### PR DESCRIPTION
**Pull request by heinezen**
Reslves https://github.com/corona-warn-app/cwa-website/issues/798

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -535,6 +535,14 @@
                             "The <a href='https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md' target='_blank' rel='noopener noreferrer'>Solution Architecture Document</a> describes the technical process in detail."
                         ]
                     },
+                    {
+                        "title": "Can I enter a test result from a Corona rapid test (Schnelltest)?",
+                        "anchor": "rapid_test",
+                        "active": true,
+                        "textblock": [
+                            "No, entering results from rapid tests is currently not supported by the Corona Warn App."
```
Comment (MikeMcC399) [inline src/data/faq.json:543]: Corona Warn App => Corona-Warn-App

In other places in the FAQ the name is used by hyphens. 
Also in the English language version of the Google App Store
https://play.google.com/store/apps/details?id=de.rki.coronawarnapp&hl=en the name uses hyphens.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
I've made two suggestions for change.

**Comment by MikeMcC399** [comment]
I thought I had added the comment about "zur Zeit" earlier, but something must have gone wrong.

File: src/data/faq_de.json [hunk 2]
```diff
@@ -544,6 +544,14 @@
                             "Das <a href='https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md' target='_blank' rel='noopener noreferrer'>Solution-Architecture-Dokument</a> beschreibt den technischen Ablauf im Detail."
                         ]
                     },
+                    {
+                        "title": "Kann ich das Ergebnis eines Corona-Schnelltests eingeben?",
+                        "anchor": "rapid_test",
+                        "active": true,
+                        "textblock": [
+                            "Nein, die Eingabe von Schnelltests wird von der Corona-Warn-App zur Zeit nicht unterstützt."
```
[the remark this answers is not in the document]
Comment (heinezen) [reply src/data/faq_de.json]: fixed

File: src/data/faq.json (hunk 1 above)
Comment (heinezen) [reply src/data/faq.json]: fixed and added a reminder to #524 so that the spelling for other entries will be fixed

**Pull request merged**